server/api/order.js

```diff
@@ -27,3 +27,22 @@ router.post('/', async (req, res, next) => {
     next(err)
   }
 })
+
+router.get('/', async (req, res, next) => {
+  try {
+    if (!req.user) {
+      throw new Error('Not available for unauthenticated users')
+    } else if (req.user.isAdmin) {
+      const orders = await Order.findAll({})
+      res.json(orders)
+    } else {
+      const {user} = req
+      const orders = await Order.findAll({
+        where: {userId: user.id}
+      })
+      res.json(orders)
```

Comment:
I think if we're committed to being RESTful here that an admin getting orders and a user getting orders would be separate routes (e.g. /api/order and /api/order/:userId) but I don't think this should hold up getting this change pushed to master. I will go ahead and approve the change.